docs/src/content/docs/fr/getting-started.mdx

```diff
@@ -93,37 +141,34 @@ Starlight est une intégration Astro, et est mis à jour comme toute intégratio
 <TabItem label="npm">
 
 ```sh
-# mettre à jour Starlight avec npm
 npm install @astrojs/starlight@latest
 ```
 
 </TabItem>
 <TabItem label="pnpm">
 
 ```sh
-# mettre à jour Starlight avec pnpm
 pnpm upgrade @astrojs/starlight --latest
 ```
 
 </TabItem>
 <TabItem label="Yarn">
 
 ```sh
-# mettre à jour Starlight avec yarn
 yarn upgrade @astrojs/starlight --latest
 ```
 
 </TabItem>
 </Tabs>
 
-Vous pouvez voir une liste complète des modifications apportées à chaque version dans [le journal des modifications de Starlight](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md).
+Consultez le [journal des modifications de Starlight](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md) pour une liste complète des modifications apportées à chaque version.
```

Comment:
Reference for the "changelog" translation: https://fr.wikipedia.org/wiki/Changelog